src/interfaces/business-interfaces.ts

```diff
@@ -51,6 +61,7 @@ export interface ApiBusinessIF {
   naicsDescription: string // firm only
   naicsKey: string // firm only
   nextAnnualReport: ApiDateTimeUtc // BCOMP only
+  operatingName?: string
```

Comment:
I'm thinking you don't need both the alternate names array and the operating name.

I think it's better to store the alternate names and compute the operating name from it (in business store, line 93). Basically, the code you have on line 298 would get moved to line 93.